sretoolbox/binaries/package.py

```diff
@@ -0,0 +1,79 @@
+# Copyright 2021 Red Hat
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+"""
+Abstractions around the kubectl-package binary.
+"""
+
+import os
+import platform
+import re
+
+from semver import VersionInfo
+
+from sretoolbox.binaries.base import Binary
+
+
+class Package(Binary):
```

Comment:
nit: Why not `KubectlPackage`? `Package` looks pretty generic and may clash easily.